index.js

```diff
@@ -187,13 +187,24 @@ export class Mixpanel {
      *     Mixpanel using the same disinct_id will be considered associated with the
      *     same visitor/customer for retention and funnel reporting, so be sure that the given
      *     value is globally unique for each individual user you intend to track.
-     *
+     * @returns {Promise} A promise that resolves when the identify is successful.
+     *     It does not return any value.
```

Comment:
Is this a breaking API change?

Reply:
it's still compatible with the old way of calling the api.